helix/pages/4_Train_Models.py

```diff
@@ -231,10 +231,14 @@ def pipeline(
         st.session_state[MachineLearningStateKeys.RerunML] = False
 
     if not already_trained_models or st.session_state[MachineLearningStateKeys.RerunML]:
-        ml_options_form()
+        # Get problem type from execution options
+        exec_opts = load_execution_options(
+            execution_options_path(helix_experiments_base_dir() / experiment_name)
+        )
```

Comment:
These are already loaded further up the page. You don't need to read the execution options again. Look at line 215